src/PS/PhaseSpace.cpp

```diff
@@ -498,6 +530,20 @@ std::string vfps::PhaseSpace::cl_code_integral = R"(
         *result = value;
     }
     )";
+std::string vfps::PhaseSpace::cl_code_integral_sum = R"(
```

Comment:
With a bit of distance from this code, I had to compare parallel to sequential code to find whether this is correct. Maybe, it is a good idea to refactor the code so that it located closer to each other. (This is valid for all OpenCL code, so this is not an issue for this PR.)